"""
15) When is the finally block executed?
==> The finally block always executes when the try block exits. 
This ensures that the finally block is executed even if an unexpected 
exception occurs.


16) What happens when „1‟== 1 is executed?
- It simply evaluates to False and does not raise any exception."""